app/views/content/life-as-a-teacher/pay-and-benefits/_block-promos.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="row">
```

Comment:
I think the name of this block promo file is too generic - I think we should make it more descriptive so it can be more easily found?